javaslang-benchmark/pom.xml

```diff
@@ -42,6 +42,11 @@
             <artifactId>jmh-generator-annprocess</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I see... you are right, that version of `org.scalaz` doesn't run with the latest `org.scala-lang`.

Is it really worth testing against `org.scalaz`, given it seems to be so highly under performing?

I personally find the comparison against the Scala implementation more meaningful, and hence I would prefer to control explicitly the Scala version. However, if the `org.scalaz` comparison is a must, then yeah, we are at the mercy of that library.

Wouldn't it be more useful to test against other implementations such as Functional Java or PCollections instead?